### ISSUE TITLE
Cryptopia

### ISSUE THREAD

**Issue by joelbrennan0** [askmike/gekko#1589]
Hi,

I'd like to put in a request to add Cryptopia Exchange if that would be suitable at any time:
https://www.cryptopia.co.nz

I'd like to put gekko on some weird and highly volotile altcoins.

Thanks

**Note: for support questions, please join our [Discord server](https://discord.gg/26wMygt)**

* **I'm submitting a ...**
[ ] bug report
[X ] feature request
[ ] question about the decisions made in the repository

* **Action taken** (what you did)

* **Expected result** (what you hoped would happen)

* **Actual result** (unexpected outcome)

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)

**Comment by Jiropole** [comment]
I might take a crack at it if I have some time in the next few days. Thanks for the clear directions @askmike. I haven't even used Gekko yet for other exchanges, so might need help testing. Anyone know the best cryptopia module? Otherwise might try the freshest: cbank/cryptopia.

**Comment by scarnago** [comment]
I'm interested on cryptopia exchange feature too

**Comment by blastbeng** [comment]
i'm interested too

**Comment by bigbruno** [comment]
And me

**Comment by HammerTron** [comment]
me too please... Has anyone started on this already?

**Comment by HammerTron** [comment]
https://www.npmjs.com/package/cryptopia-api

**Comment by Jiropole** [comment]
I spent a couple hours on this a few weeks ago, but got distracted on other projects. I did have a discussion with Mike about making `Trader.getCapabilities` asynchronous / dynamic, however it sounds like this would interfere with stability / speed requirements. While I didn't get far, I offer you my unfinished project for harvesting that data to be hardcoded into Gekko: https://github.com/Jiropole/get-cryptopia-pairs. I believe it harvests assets correctly, but pairs are unfinished as minimalOrder needs to be populated.
